File: classlink-oneroster-oauth1/java/src/com/classlink/roster/OneRosterResponse.java
package com.classlink.roster;

public class OneRosterResponse {
    private int statusCode;
    private String response;

    public int getStatusCode() {
        return statusCode;
    }

    public String getResponse() {
        return response;
    }

    public OneRosterResponse(int statusCode, String response) {
        this.statusCode = statusCode;
        this.response = response;
    }
}

File: classlink-oneroster-oauth1/dotnet/OneRoster/OneRoster/OneRosterResponse.cs
namespace ClassLink.OneRoster
{
    public class OneRosterResponse
    {
        public int StatusCode { get; }
        public string Response { get; }

        public OneRosterResponse(int statusCode, string response)
        {
            StatusCode = statusCode;
            Response = response;
        }
    }
}